gen/enum_test.go

```diff
@@ -335,6 +335,28 @@ func TestUnmarshalTextReturnsErrorOnInvalidValue(t *testing.T) {
 	assert.Error(t, err)
 }
 
+func TestMarshalTextReturnsValue(t *testing.T) {
+	tests := []struct {
+		title string
+		e     te.EnumDefault
+		text  string
+	}{
+		{`0 <-> Foo`, te.EnumDefaultFoo, "Foo"},
+		{`1 <-> Bar`, te.EnumDefaultBar, "Bar"},
+		{`2 <-> Baz`, te.EnumDefaultBaz, "Baz"},
+		{`42 <-> 42`, 42, "42"},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.title, func(t *testing.T) {
+			b, err := tt.e.MarshalText()
+			if assert.NoError(t, err, "unable to marshal enum") {
```

Comment:
optional: can `require.NoError` and skip the if statement, especially if you add the UnmarshalText verification.

Reply:
Thanks for the heads up. I'll leave it as is since the rest of the file seems to be using if statements as well